Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index-v8.md

```diff
@@ -0,0 +1,270 @@
+---
+versionFrom: 8.0.0
+needsV9Update: "true"
```

Comment:
Is "needsV9Update" still required? I see an index.md file for v9...

Reply:
No you are right, i'll remove it :D